.travis.yml

```diff
@@ -0,0 +1,30 @@
+dist: trusty
+sudo: false
+language: python
+matrix:
+  include:
+    - env: TOXENV=py36
+      python: 3.6
+    - language: ruby
+      rvm: 2.5
```

Comment:
About this part, we can add Python 3.7, Ruby 2.2, 2.3, 2.4 later on another pull-request.